data/json/mapgen/s_lightindustry.json

```diff
@@ -173,6 +173,7 @@
         "3": "f_air_compressor",
         "4": "f_drill_press",
         "9": "f_heavy_lathe",
+        "K": "f_beverly_shear",
```

Comment:
It's defined here, but doesn't seem to be placed anywhere?

Reply:
Good catch, thanks! Latest commit replaces two of the light machinery furnitures with the Beverly shear.